ses

```diff
@@ -65,9 +65,22 @@ if [ -x /usr/bin/ceph ]; then
     # remain, because their output is easier to read in a hurry ;)
     plugin_command "ceph --connect-timeout=$CT report" > $LOG/ceph/ceph-report 2>&1
     plugin_command "timeout $CT rados df" > $LOG/ceph/rados-df 2>&1
-    plugin_command "timeout $CT rbd ls" > $LOG/ceph/rbd-ls 2>&1
     plugin_command "radosgw-admin period get" > $LOG/ceph/radosgw-admin-period-get 2>&1
 
+    timeout $CT ceph osd pool ls detail 2>/dev/null |
+    sed -nEe "s/^.*'([^']+)'.* application rbd/\\1/p" |
+    while read pools; do
+        mkdir -p $LOG/ceph/images/$pools
+        timeout $CT rbd ls $pools 2>/dev/null \
+		> $LOG/ceph/images/rbd-ls-${pools} 2>&1 |
```

Comment:
I mean this `|` here. It looks like a typo?

Reply:
it is indeed a typo, it should not be there. But that did not help so I switched to using tee instead.